linchpin/shell/__init__.py

```diff
@@ -208,8 +208,9 @@ def init(ctx):
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
 @click.option('-if', '--inventory-format', default="cfg",
               help="Inventory format can be cfg or json")
+@click.option('--ask-vault-pass', is_flag=True)
 @pass_context
-def up(ctx, targets, run_id, tx_id, inventory_format):
+def up(ctx, targets, run_id, tx_id, inventory_format, ask_vault_pass):
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.